### ISSUE TITLE
Picture rotated

### ISSUE THREAD

**Issue by jberkey78** [entaq/floq#13]
I am the only one to experience this, but when I take a picture in portrait mode with main camera while in the app it saves it on its side.  If I take the photo portrait with front camera (selfie) it is oriented correctly and when in landscape as well.

**Comment by codesworth** [comment]
Noted. Let's make this an issues to look out for from other testers. I tired couple of times but couldn't reproduce same results. So let's be on the watch for this.

**Comment by jberkey78** [comment]
yeah, one of the people at work, his profile picture that he manually added rotated on its own too.

**Comment by jberkey78** [comment]
Another user has experienced the photo rotation issue

UID: HnVGT9Br8eNkBmuOb1Id18k3A692
Email:  abukuras@gmail.com

**Comment by codesworth** [comment]
Can we please confirm the specific iOS devices that this has happened on. 

And also, this is bug only experienced when setting a new profile photo right.

**Comment by jberkey78** [comment]
Both iPhone 10's and v 12.1 although I updated to 12.1.2 after the fact and still experience issue

**Comment by jberkey78** [comment]
I wonder if it has anything to do with the size of the photos on the iPhone X?  Meaning that it's height makes the app think it is a landscape rather than a portrait.  Just a thought.

**Comment by jberkey78** [comment]
![image capture crop](https://user-images.githubusercontent.com/13186530/51347536-5e294b00-1a6e-11e9-9ffb-0608fb34364f.PNG)
![image capture default](https://user-images.githubusercontent.com/13186530/51347537-5e294b00-1a6e-11e9-84c8-f108f1d41284.PNG)

**Comment by jberkey78** [comment]
Okay so I was just playing around and my images were coming in side ways, so I took another one and cropped its height just a a little bit and it loaded correctly so it has to be something with the height.  Above are screen shots of a original and a cropped...the cropped one comes in normal the original comes in sideways. @entaq @codesworth  thoughts on this
